js/counties.js

```diff
@@ -119,7 +119,33 @@ function displayCountyBounds(error, activeCountyData) {
         unhighlightCircle();
         hideToolTip();
       })
-      .on('click', zoomToFromState);
+      .on('click', function(d) {
+        
+        // clicking a county on mobile has no affect on the 
+        // view unless it's the same county as last time
+        if(waterUseViz.mode === "mobile") {
+          
+          var prevCounty = waterUseViz.prevClickCounty;
+          var thisCountyID = d3.select(this).attr("id");
+          if(prevCounty === thisCountyID) {
+            
+            //only zoom out if you click on the same county 
```

Comment:
So I don't think we have a choice to make tapping any county zoom you out. On mobile, you need to tap a county in order for it's data to display which was the major motivation for making this change. So, if you could click anywhere and it zoomed you out, you would never be able to actually get other county information. On desktop, you can just hover so that's not an issue. 

On mobile, the dropdown and zoom out buttons are available for counties that are too small to tap. I honestly don't know if there is a better way to do the zooming and county select w/ info behavior on mobile unless we completely remove the ability to click on the map and force people to use the dropdown from the get-go.

Reply:
I agree, it shouldn't be that you can tap anywhere to zoom out. I was thinking instead that your only option for zooming out would be to use the Zoom Out button. I think if that were the only option, then users would experience it as cheaper to try and try again to shift focus to the county they're trying to see (because they wouldn't ever get zoomed out accidentally when they were just trying to focus on the tiny county they wanted)

Reply:
Jordan talked with me a bit about this today and we think that scenario could be an edge case. I think it would be more frustrating as a user to be able to tap to zoom in, but not to zoom back out. I'd like to leave as-is for now and revisit once we make a push to beta so we can all play around on mobile and try to see what still feels clunky.

Reply:
Works for me. Thanks for thinking & talking it through.